client/src/charts/canada/canada.js

```diff
@@ -0,0 +1,213 @@
+
+import {
+  formats,
+  run_template,
+  businessConstants,
+  declarative_charts,
+  NivoResponsiveHBar,
+} from "../../panels/panel_declarations/shared.js";
```

Comment:
Please break these up and import each of them from their original source rather than from the `panel_declarations/shared.js` re-exporter. More transparent about where dependencies come from that way, and less likely to result in circular dependencies, etc.

`shared.js` should only be used locally to the panel declarations (and even then I'd like to eventually get rid of the pattern).